src/Symfony/Component/Console/Tester/TesterTrait.php

```diff
@@ -91,6 +117,53 @@ public function setInputs(array $inputs)
         return $this;
     }
 
+    /**
+     * Initialize the output property.
+     *
+     * Available options:
+     *
+     *  * decorated:                 Sets the output decorated flag
+     *  * verbosity:                 Sets the output verbosity flag
+     *  * capture_stderr_separately: Make output of stdOut and stdErr separately available
+     *
+     * @param array $options An array of options
+     *
+     * @return self
```

Comment:
this one is not true and returning void is fine. Both `@param` and `@return` annotations should be removed as they do not provide any information that is not already provided by the method signature.